conf.yaml

```diff
@@ -1548,6 +1548,26 @@ contents:
                 path:   docs
                 exclude_branches:   [ 7.11, 7.10, 7.9, 7.8, 7.7, 7.6, 7.5, 7.4, 7.3, 7.2, 7.1, 7.0, 6.8, 6.7, 6.6, 6.5, 6.4, 6.3, 6.2, 6.1, 6.0, 5.6, 5.5, 5.4, 5.3, 5.2, 5.1, 5.0 ]
 
+          - title:      Fleet Developer Guide
+            prefix:     en/fleet/developer
```

Comment:
Ahhh. Makes me wish we had prefixed the user guide in the same way, `en/fleet/user`. Oh well.

Reply:
With that kind of planning, we wouldn't be able to call the build Frankie for Frankenstein.

Reply:
😅